src/vs/editor/browser/controller/mouseHandler.ts

```diff
@@ -404,7 +404,7 @@ class MouseDownOperation extends Disposable {
 
 		const options = this._context.configuration.options;
 
-		if (!options.get(EditorOption.readOnly)
+		if (true //!options.get(EditorOption.readOnly)
```

Comment:
Is this an oversight?

Reply:
Yes. 
While grep-ing for isReadonly() if found that;
and that somehow slipped into the merge.


See: #161717

Reply:
by the way, I also had code to integrate the 'oldReadonly' with the check in `updateLastResolvedFileStat`
(basically: just call `isReadonly();` at the end of `updateLastResolvedFileStat` to check & fire.

I took it out of this mod, to keep it simple.

(also because it seems that files that are detected read-only from filestat do not show the 'lock' decoration,
and I didn't want to get blamed for that...)